### PR TITLE
add test plugin install --disable and volume ls

### PR DESCRIPTION
**Pull request by vieux**
add a test for #27924 

ping @anusha-ragunathan @tiborvass

### CODE REVIEW

**Comment by anusha-ragunathan** [comment]
LGTM

File: integration-cli/docker_cli_plugins_test.go [hunk 1]
```diff
@@ -108,6 +108,16 @@ func (s *DockerSuite) TestPluginInstallDisable(c *check.C) {
 	c.Assert(strings.TrimSpace(out), checker.Contains, pName)
 }
 
+func (s *DockerSuite) TestPluginInstallDisableVolumeLs(c *check.C) {
+	testRequires(c, DaemonIsLinux, ExperimentalDaemon, Network)
+	out, _, err := dockerCmdWithError("plugin", "install", "--grant-all-permissions", "--disable", pName)
+	c.Assert(err, checker.IsNil)
+	c.Assert(strings.TrimSpace(out), checker.Contains, pName)
+
+	_, _, err = dockerCmdWithError("volume", "ls")
```
Comment (cpuguy83) [inline integration-cli/docker_cli_plugins_test.go]: Why not `dockerCmd("volume", "ls")`

File: integration-cli/docker_cli_plugins_test.go [hunk 2]
```diff
@@ -108,6 +108,16 @@ func (s *DockerSuite) TestPluginInstallDisable(c *check.C) {
 	c.Assert(strings.TrimSpace(out), checker.Contains, pName)
 }
 
+func (s *DockerSuite) TestPluginInstallDisableVolumeLs(c *check.C) {
+	testRequires(c, DaemonIsLinux, ExperimentalDaemon, Network)
+	out, _, err := dockerCmdWithError("plugin", "install", "--grant-all-permissions", "--disable", pName)
```
Comment (cpuguy83) [inline integration-cli/docker_cli_plugins_test.go]: Why not `dockerCmd("volume", "ls")`

**Review by cpuguy83 — CHANGES_REQUESTED** [review]

**Comment by AkihiroSuda** [comment]
LGTM

**Review by vdemeester — APPROVED** [review]
LGTM once @cpuguy83 comments taken care of 🐸

File: integration-cli/docker_cli_plugins_test.go (hunk 1 above)
Comment (vieux) [inline integration-cli/docker_cli_plugins_test.go]: because we want to check the error ?

Comment (vdemeester) [inline integration-cli/docker_cli_plugins_test.go]: @vieux yeah but you check that `err` is nil, that what `dockerCmd("volume", "ls")` basically does 👼

**Comment by vieux** [comment]
@cpuguy83 PTAL

File: integration-cli/docker_cli_plugins_test.go [hunk 3]
```diff
@@ -108,6 +108,15 @@ func (s *DockerSuite) TestPluginInstallDisable(c *check.C) {
...
```
Comment (cpuguy83) [inline integration-cli/docker_cli_plugins_test.go]: Same with this one, can just be `dockerCmd`

**Review by cpuguy83 — APPROVED** [review]
LGTM

File: integration-cli/docker_cli_plugins_test.go (hunk 3 above)
Comment (vieux) [inline integration-cli/docker_cli_plugins_test.go]: here I'm still checking the output for consistency

Comment (cpuguy83) [inline integration-cli/docker_cli_plugins_test.go]: Yep, `out, (exitcode) := dockerCmd(...)`

**Pull request merged**